python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/qos.schema.yml

```
@@ -57,3 +57,10 @@ keys:
                   chip_based:
                     type: bool
                     description: Allow non-ect chip-based.
+      tx_queue:
+        type: dict
+        description: Global txQueue settings.
```

Following EOS help would it make more sense to use:

```suggestion
        description: Global transmit queue settings.
```

```
site2-wan2#show qos ?
  interfaces    Show QoS status for a specific interface
  map           Mapping Configuration of different QoS parameters
  maps          Show various QoS mappings
  profile       Show QoS profile
  subinterface  Subinterface info
  tx-queue      Show transmit queue global configuration <<<<<<<<<<<<<<
```